book/integration-service.md

```diff
@@ -0,0 +1,207 @@
+# Integration Service
+
+
+## Overview
+
+The Integration Service is composed of controllers that facilitate automated testing of content produced by the build pipelines. It is mainly responsible for integration testing capabilities.
+
+The Integration service uses the pipeline, snapshot, and environment controllers to watch for Component builds, trigger and manage testing Tekton Pipelines, and create releases based on the testing outcome(s).
+
+### Goals
+
+- The Integration service need to be able to deploy a specific set of images to an environment and record the results of integration testing.
+- Given a specific set of images, Integration service should be able to tell if they passed application validation tests.
+- The Integration service should be able to update one component image at a time in order to support 'slow loop' application validation testing.
+- The Integration service should have a capability to automatically promote sets of passed images.
+- To be SLSA compliant, Integration service should also be able to tell what version of the build and test pipelines were used to create each component image.
```

Comment:
You are using SLSA without defining it. Can you at least link to some reference on SLSA compliance?

Reply:
Done